src/EnergyPlus/CMakeLists.txt

```diff
@@ -757,6 +757,7 @@ target_link_libraries(
          DElight
          libkiva
          btwxt
+         courierr
```

Comment:
The target link didn't need to be here, so it's been removed. But, courierr is Big Ladder's current lightweight error-logging interface. The idea is that a client user would provide a concrete implementation of the courierr interface as an injected dependency, rather than an old C-style callback, to handle logging.